.gitignore

```diff
@@ -80,7 +80,6 @@ Tools/MonoGame.Build.Tasks/MonoGame.Build.Tasks.*.csproj
 ThirdParty/Lidgren.Network/*.csproj
 Test/*.csproj
 !Test/MonoGame.Test.XNA.csproj
-/*.sln

```

Comment:
Don't remove this line from gitignore, we still need it, instead just force add .sln file you need.